data/abilities.js

```diff
@@ -2462,13 +2462,9 @@ let BattleAbilities = {
 	"prankster": {
 		shortDesc: "This Pokemon's Status moves have priority raised by 1, but Dark types are immune.",
 		onModifyPriority: function (priority, pokemon, target, move) {
-			if (move && move.category === 'Status') {
-				return priority + 1;
-			}
-		},
-		onModifyMove: function (move) {
 			if (move && move.category === 'Status') {
 				move.pranksterBoosted = true;
+				return priority + 1;
```

Comment:
Nice, this should also fix the Dark + Prankster + Encore bug (https://www.smogon.com/forums/posts/7715202).

Reply:
Oh, I didn't see you'd already linked that, sorry.